R/parking_meters.R

```diff
@@ -0,0 +1,37 @@
+#' Vancouver Parking Meters
+#'
+#' This dataset contains information on the rates and time limits for parking meters in Vancouver. Information is shown for entire block faces rather than for individual meters. Data in the portal is updated weekly.
+#'
+#' @source <https://opendata.vancouver.ca/explore/dataset/parking-meters/information/>, downloaded 2020-05-06
+#' @format A data frame with columns:
+#' \describe{
+#'  \item{meter_head}{Meter head type.}
+#'  \item{r_mf_9a_6p}{Rate for Mon-Fri 9AM to 6PM.}
+#'  \item{r_mf_6p_10}{Rate for Mon-Fri 6PM to 10PM.}
+#'  \item{r_sa_9a_6p}{Rate for Sat 9AM to 6PM.}
+#'  \item{r_sa_6p_10}{Rate for Sat 6PM to 10PM.}
+#'  \item{r_su_9a_6p}{Rate for Sun 9AM to 6PM.}
+#'  \item{r_su_6p_10}{Rate for Sun 6PM to 10PM.}
+#'  \item{rate_misc}{Miscellaneous rate information.}
+#'  \item{time_in_effect}{Times that meter is in effect.}
```

Comment:
the `time_in_effect` currently has the format `METER IN EFFECT: 9:00 AM TO 10:00 PM`, should I split this into two columns like `in_effect_from` and `in_effect_to`?

Reply:
I think for tidy data, the current column makes the most sense.